docs/site/development/development-process.md

```diff
@@ -0,0 +1,292 @@
+| dsub | The ticket should be completed before submission starts. |
+| dvet | The ticket should be completed before vetting starts |
+| rc  | The ticket must be completed before **Alpha — Final Data candidate**. No change to data affecting ICU thereafter. Other dtd, data, spec, docs, tool changes allowed. |
+| spec-beta | The ticket must be completed before **Beta — Final Candidate**. No dtd or data changes allowed thereafter. Docs, charts, spec changes allowed (= ICU release candidate) |
+| final | The ticket must be completed before **Release** ( = ICU Release) |
+
+### Assignees
+
+   1.  Assignees periodically review and take care of their tickets.
+        
+        1.  Go to [http://unicode.org/cldr/trac/report/61](http://www.google.com/url?q=http%3A%2F%2Funicode.org%2Fcldr%2Ftrac%2Freport%2F61&sa=D&sntz=1&usg=AOvVaw0UYmG2ygYOMySNUAGnbPKu) for **My Tickets**
+            
+
+      1.  **design:** ~~You must go back to the committee with the completed design before checking anything in. For example, if the ticket doesn't describe an exact DTD structure, then add the proposed structure to the ticket in a comment, and send an email to the committee asking whether anyone objects.~~
+    
+3.  **assess:** ~~Tickets marked "assess" need to be evaluated for priority (or whether to do them at all). Send your suggested disposition to the cldr group by email.~~
+    
+    1.  **duplicate:** If the ticket is a duplicate, any extra information is copied to the other ticket, an xref is set in both, and the ticket is returned.
+        
+    2.  **splitting:**
+        
+        1.  If the ticket requires separate phases, such as a change to the spec, then a separate ticket is filed, with xrefs set in both.
+                
+
+        2.  The owner can carry over all the triage acceptance including Owner, Component, etc.
+            
+            1.  If part of a ticket cannot be completed in the release:
+                
+                1.  Set the milestone to the release number
+                    
+                2.  Add a comment indicating what part of the ticket was done, and what part wasn't.
+                    
+                3.  Set a reviewer.
+                    
+                4.  Create a new ticket for the remaining work, with an xref to the old bug.
+                    
+
+    3.  **problems:**
+        
+
+        1.  If the ticket can't be done due to lack of time, or is being downgraded in priority from major or above to below major, send an email to the cldr group.
+                
+        2.  If during the course of doing the ticket, the person runs up against a problem, then send a message to the cldr group. Examples:
+                
+
+            *   The description doesn't work
+                
+            *   Another mechanism looks better
+                
+
+### DTD Changes
+
+*   **If you are making any DTD changes, please follow the instructions on** [**Updating DTDs**](/development/updating-dtds)**.**
```

Comment:
```suggestion
*   **If you are making any DTD changes, please follow the instructions on** [**Updating DTDs**](https://cldr.unicode.org/development/updating-dtds)**.**
```

site-relative link (and there are others) but perhaps it's better to just make them absolute to start and then we can fix up en masse later?

Reply:
Yes, good idea.